specification/metadata/schema/providerversion.md

```diff
@@ -0,0 +1,30 @@
+# Provider Version
+
+The ProviderVersion MAY be provided to allow the provider to declare the version of logic by which the FOCUS dataset was generated and is separate from Focus Version. ProviderVersion allows for the provider to specify changes that may not result in a structural change in the data. It is suggested that the provider version use a versioning approach such as [SemVer](https://semver.org) version.
```

Comment:
To keep with the pattern we've been using, this needs a first paragraph that's more descriptive and a second paragraph can be the normative prose. In this case, if we have a requirements section similar to FOCUS_Spec/specification/attributes/key_value_format.md.

We use this language consistently:
"All key-value related columns defined in the FOCUS specification MUST follow the key-value formatting requirements listed below."

Reply:
wow that is a lot of use of the word provide....

Reply:
nit: Focus Version or FOCUS Version?